src/services/__mocks__/update-todo.service.ts

```diff
@@ -0,0 +1,4 @@
+import { Todo } from '@/database/entities/__mocks__/todo.entity';
+export class UpdateTodoService {
+  update = jest.fn(() => Todo);
```

Comment:
![Screenshot_317](https://user-images.githubusercontent.com/32253743/107866062-949f4300-6e4b-11eb-9f25-fd0865195618.png)

Is this really necessary? Why not combine it all into one file? Please review your __mocks__ folders.

Reply:
If I group everything, this class will be a "god class", so I will violate SRP.